package.json

```diff
@@ -107,7 +107,7 @@
     "build-dist": "bash ./scripts/build-dist.sh",
     "build-win-installer": "scripts\\build-windows-installer.bat",
     "dupe-check": "yarn jsinspect ./src",
-    "lint": "eslint . && flow check",
+    "lint": "eslint src __tests__ && flow check",
```

Comment:
why did you change this?

Reply:
It was for when there was no flow types, I'll revert it